services/web/server/tests/unit/with_dbs/02/test_projects_handlers__open_close.py

```diff
@@ -591,6 +591,7 @@ def one_max_open_studies_per_user(
         )
 
 
+@pytest.mark.testit
```

Comment:
Please remind me, does this mean the test will be skipped? I find it kind of counter intuitive if that's the case.

Reply:
no. this marker is mostly while developing. Should not be commited. I forgot here.